lib/wallet/src/Cardano/Wallet/DB/Layer.hs

```diff
@@ -526,16 +535,16 @@ newDBLayerWith _cacheBehavior _tr ti SqliteContext{runQuery} = do
             case fromGenesis cp of
                 Nothing -> throwIO $ ErrInitializeGenesisAbsent wid header
                 Just wallet ->
-                    updateDBVar walletsDB_ $ Insert wid wallet
+                    updateDBVar walletsDB__ $ Insert wid wallet
```

Comment:
If I don't understand this naming convention I'll have trouble following it...

Reply:
I have renamed it to `walletsDB`, which became possible by removing the field import for `DBLayer`.